_posts/2017-10-03-coworking.markdown

```diff
@@ -0,0 +1,9 @@
+---
+title: Tuesday Coworking Session
+text: Work through any of your coding problems in a friendly and collaborative environment.
+location: MP408
```

Comment:
Ah, decided to stick with MP408 eh. No other area that we could find? I guess we can change it if/when we find something more 'cozy'/'comfortable'.

Reply:
I think that's a typo; the event posting says HH reading room.

Reply:
sorry, thanks! typo indeed